src/qonlinetts.cpp

```diff
@@ -173,8 +299,9 @@ QString QOnlineTts::languageApiCode(QOnlineTranslator::Engine engine, QOnlineTra
     switch (engine) {
     case QOnlineTranslator::Google:
     case QOnlineTranslator::Lingva: // Lingva is a frontend to Google Translate
-        if (lang != QOnlineTranslator::Auto)
-            return QOnlineTranslator::languageApiCode(engine, lang); // Google use the same codes for tts (except 'auto')
+        if (lang != QOnlineTranslator::Auto) {
+            return regionCode(lang, m_regionPreferences.value(lang)); // Google use the same codes for tts (except 'auto')
+        }
```

Comment:
We usually don't put braces around single statements:
```suggestion
        if (lang != QOnlineTranslator::Auto)
            return regionCode(lang, m_regionPreferences.value(lang)); // Google use the same codes for tts (except 'auto')
```